docs/document/i18n/ja/docusaurus-plugin-content-docs/current/components/desktop/dialog.md

```diff
@@ -1,20 +1,16 @@
 ---
-id: version-1.11.0-dialog
+id: dialog
 title: Dialog
 sidebar_label: Dialog
-original_id: dialog
 ---
 
 ## Overview
 
 Dialog は、ダイアログボックスを表示します。
 
-<div class="sample-container" id="dialog">
-  <div id="sample-container__components">
-    <iframe id="iframe" title="dialog image" width="700px" height="300px"></iframe>
-  </div>
-</div>
-<script src="/js/samples/desktop/dialog.js"></script>
+import { DialogComponent } from "@site/static/js/samples/desktop/dialog.js"
+
+<DialogComponent />
```

Comment:
Dialog component preview of JA font seems wrong.

Reply:
Yes, because on this page , we used ja font
You can check live website with other component
ex: font of button
- https://kintone-ui-component.netlify.app/docs/en/components/desktop/button
- https://kintone-ui-component.netlify.app/docs/ja/components/desktop/button

Reply:
Got it! I confirmed it's ok.